### ISSUE TITLE
Depot Overview crashes after select a Train

### ISSUE THREAD

**Issue by SkiperTheBoss** [Yousei9/Logistic-Train-Network#169]
*Bug description & to Reproduce*
When i open the Depot Overview and click on a Train of the Train list it just crash after i pressed.

*Error message*
```
1527.421 Error MainLoop.cpp:1142: Exception at tick 17266419: The mod LTN Tracker caused a non-recoverable error.
Please report this error to the mod author.

Error while running event LTN_Tracker::on_gui_click (ID 1)
__LTN_Tracker__/ltnt/util.lua:12: attempt to call field 'get_main_locomotive' (a nil value)
stack traceback:
	__LTN_Tracker__/ltnt/util.lua:12: in function 'select_train'
	__LTN_Tracker__/ltnt/gui_ctrl.lua:314: in function '?'
	__LTN_Tracker__/ltnt/gui_ctrl.lua:191: in function <__LTN_Tracker__/ltnt/gui_ctrl.lua:179>
```

EDIT:
After updating to 0.17.24 i got with the time this error:
```
Exception at tick 17513943: The mod LTN - Logistic Train Network caused a non-recoverable error.
Please report this error to the mod author.

Error while running event LogisticTrainNetwork::on_tick (ID 0)
The mod LTN Tracker caused a non-recoverable error.
Please report this error to the mod author.

Error while running event LTN_Tracker::Custom event (ID 163)
__LTN_Tracker__/ltnt/data_processing.lua:493: attempt to call upvalue 'get_main_loco' (a nil value)
stack traceback:
	__LTN_Tracker__/ltnt/data_processing.lua:493: in function <__LTN_Tracker__/ltnt/data_processing.lua:486>
stack traceback:
	__LogisticTrainNetwork__/script/dispatcher.lua:87: in function <__LogisticTrainNetwork__/script/dispatcher.lua:34>
stack traceback:
	[C]: in function 'raise_event'
	__LogisticTrainNetwork__/script/dispatcher.lua:87: in function <__LogisticTrainNetwork__/script/dispatcher.lua:34
```

And here is 2 Screenshot which should be explain themselves.

The names are always the same:
![LTN_Tracker_Delivery_History](https://user-images.githubusercontent.com/39216046/55433055-c8915800-5594-11e9-8f27-55debb83780f.jpg)

Those resources are all on different train stops but they all show only on one:
![LTN_Tracker_Stations](https://user-images.githubusercontent.com/39216046/55433056-c8915800-5594-11e9-81e7-90d81dc601d7.jpg)

Version 0.17.23
Greetings Skiper

**Comment by SkiperTheBoss** [comment]
Okay problems with the Screenshot got solved with 0.17.24

**Comment by Yousei9** [comment]
That's an issue with LTN tracker in the first place.

**Comment by eduran84** [comment]
It is fixed in version 0.9.9 of LTNT. See also: https://github.com/eduran84/LTN_Tracker/issues/21
